Xamarin.Essentials/Launcher/Launcher.ios.tvos.cs

```diff
@@ -30,47 +31,27 @@ static Task<bool> PlatformTryOpenAsync(Uri uri)
         }
 
 #if __IOS__
+
         static UIDocumentInteractionController documentController;
 
         static Task PlatformOpenAsync(OpenFileRequest request)
         {
-            var fileUrl = NSUrl.FromFilename(request.File.FullPath);
-
-            documentController = UIDocumentInteractionController.FromUrl(fileUrl);
-            documentController.Delegate = new DocumentControllerDelegate
+            documentController = new UIDocumentInteractionController()
             {
-                DismissHandler = () =>
-                {
-                    documentController?.Dispose();
-                    documentController = null;
-                }
+                Name = request.File.FileName,
+                Url = NSUrl.FromFilename(request.File.FullPath),
+                Uti = request.File.ContentType
             };
-            documentController.Uti = request.File.ContentType;
-
-            var vc = Platform.GetCurrentViewController();
 
-            CoreGraphics.CGRect? rect = null;
-            if (DeviceInfo.Idiom == DeviceIdiom.Tablet)
-            {
-                rect = new CoreGraphics.CGRect(new CoreGraphics.CGPoint(vc.View.Bounds.Width / 2, vc.View.Bounds.Height), CoreGraphics.CGRect.Empty.Size);
-            }
-            else
-            {
-                rect = vc.View.Bounds;
-            }
-
-            documentController.PresentOpenInMenu(rect.Value, vc.View, true);
+            var view = Platform.GetCurrentUIViewController().View;
+            var rect = DeviceInfo.Idiom == DeviceIdiom.Tablet
+                ? new CGRect(new CGPoint(view.Bounds.Width / 2, view.Bounds.Height), CGRect.Empty.Size)
+                : view.Bounds;
```

Comment:
Not for this PR, but maybe we should think of adding a `PresentationSourceBounds` property to file request so that the user can potentially control this. Could you open an issue for that?

Reply:
Ок

Reply:
https://github.com/xamarin/Essentials/issues/1611

Reply:
Awesome, thanks.